app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -157,39 +157,38 @@ class SettingsActivity :
     private fun configureUiEventHandlers() {
         with(viewsGeneral) {
             setAsDefaultBrowserSetting.setOnCheckedChangeListener(defaultBrowserChangeListener)
-            homeScreenWidgetSetting.setOnClickListener { viewModel.userRequestedToAddHomeScreenWidget() }
+            homeScreenWidgetSetting.setClickListener { viewModel.userRequestedToAddHomeScreenWidget() }
         }
 
         with(viewsAutofill) {
-            autofill.setOnClickListener { viewModel.onAutofillSettingsClick() }
+            autofill.setClickListener { viewModel.onAutofillSettingsClick() }
         }
 
         with(viewsAppearance) {
             selectedThemeSetting.setOnClickListener { viewModel.userRequestedToChangeTheme() }
-            changeAppIconLabel.setOnClickListener { viewModel.userRequestedToChangeIcon() }
+            changeAppIconLabel.setClickListener { viewModel.userRequestedToChangeIcon() }
             selectedFireAnimationSetting.setOnClickListener { viewModel.userRequestedToChangeFireAnimation() }
-            accessibilitySetting.setOnClickListener { viewModel.onAccessibilitySettingClicked() }
+            accessibilitySetting.setClickListener { viewModel.onAccessibilitySettingClicked() }
         }
 
         with(viewsPrivacy) {
             globalPrivacyControlSetting.setOnClickListener { viewModel.onGlobalPrivacyControlClicked() }
             autoconsentSetting.setOnClickListener { viewModel.onAutoconsentClicked() }
-            fireproofWebsites.setOnClickListener { viewModel.onFireproofWebsitesClicked() }
+            fireproofWebsites.setClickListener { viewModel.onFireproofWebsitesClicked() }
             automaticallyClearWhatSetting.setOnClickListener { viewModel.onAutomaticallyClearWhatClicked() }
-            automaticallyClearWhenSetting.setOnClickListener { viewModel.onAutomaticallyClearWhenClicked() }
```

Comment:
This was missed:
```
automaticallyClearWhenSetting.setOnClickListener { viewModel.onAutomaticallyClearWhenClicked() }
```

Reply:
@malmstein I think the above comment got lost :) There is a missing click listener:
`automaticallyClearWhenSetting.setOnClickListener { viewModel.onAutomaticallyClearWhenClicked() }`